hm_pyhelper/hardware_definitions.py

```diff
@@ -38,7 +38,10 @@ def is_raspberry_pi():
         'BUTTON': 26,
         'ECCOB': True,
         'TYPE': 'Full',
-        'CELLULAR': False
+        'CELLULAR': False,
+        'FCC_IDS': ['2AZDM-HNTIN'],
+        'CONTAINS_FCC_IDS': ['2AHRD-EPN8531', '2AB8JCSR40', '2ARPP-GL5712UX'],
+        'ICC_IDS': ['27187-HNTIN']
```

Comment:
should be IC or ISED really

Reply:
Don't know this type TBH, is this right? https://www.ic.gc.ca/eic/site/ceb-bhst.nsf/eng/h_tt00051.html#Q10

Reply:
yep lets go with IC